lib/order.rb

```diff
@@ -0,0 +1,33 @@
+require 'menu'
+
+class Order
+  def initialize(*choice, quantities, user_total)
```

Comment:
This seems like a lot of arguments to have to pass in, and a lot of work that initialize is doing. Maybe you could split it up between the methods, e.g. add items/quantities one by one through a separate method and then get the user_total after that.